upgrade/upgrade-scripts/common.sh

```diff
@@ -38,8 +39,13 @@ function get_image_version() {
 	basename "$(get_image_path)"
 }
 
+function get_platform() {
+	dpkg-query -Wf '${Package}' 'delphix-entire-*' | \
```

Comment:
Please verify this works for not-in-place upgrades; e.g.
```
$ sudo /var/dlpx-update/latest/upgrade -v not-in-place
```
when running `execute` for that upgrade type, `delphix-entire` isn't installed yet, so I don't think doing this `dpkg-query` will return anything useful.

Reply:
I've modified the `execute` script to have the platform passed in. That seems a lot easier than trying to determine the platform from withing the container. Let me know what you think.

Reply:
Yea, that seems reasonable to me. Thank you!

Reply:
Thinking about this a little more, do we need to do anything special when we call `debootstrap` w.r.t. the platform? e.g. I'm curious which kernel package it'll install by default, and/or if it'll install a kernel package at all...

Reply:
I think `debootstrap` doesn't install any kernel packages (at least the way we are using it).

Reply:
Sweet. If it just works, that's great! Thanks.